java-frontend/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -59,7 +61,9 @@ public void checkEndOfExecutionPath(CheckerContext context, ConstraintManager co
   @Override
   public void scanFile(JavaFileScannerContext context) {
     for (SEIssue seIssue : issues) {
-      context.reportIssueWithFlow(this, seIssue.getTree(), seIssue.getMessage(), seIssue.getFlows(), null);
+      // FIXME SONARJAVA-2111 report all flows
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)